### PR TITLE
Cailey/bugfix/sa fixes

### PR DESCRIPTION
**Pull request by caggles**
- Splits makefile deployment up based on privileges
- Moves archeobot out of devops-artifactory and into devops-archeobot namespace
- Recover a couple of missing changes present in the current version of the operator
- Switch from using a local within-namespace URL for artifactory to an external URL
- Re-adds test folder (don't worry about closely reviewing these files - they're old, but in place to basically serve as a template for adding properly updated testing files later)
- Re-adds CR templates
- Removes steps for including the subdomain docker secret, adds steps for including the repo path docker secret

### CODE REVIEW

File: archeobot/test.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - community.kubernetes
+    - operator_sdk.util
+
+  tasks:
+
+  - name: Check artifactory user
+    uri:
+      url: "https://artifacts.apps.klab.devops.gov.bc.ca/artifactory/api/security/users/test1"
+      user: "admin"
+      password: "8FEiW44h7V6IpjyS6wiQcJ6PC7s2ko"
```
Comment (caggles) [inline archeobot/test.yaml]: i'm dumb, i thought i'd removed this already. i changed the password, lol.

**Review by sbathgate — APPROVED** [review]
Odd, it originally showed only two readme edits haha. Will do a proper review now.

**Review by sbathgate — APPROVED** [review]
Overall, there is admittedly a fair bit here that is still a bit out of the scope of my knowledge to fully analyze. With that in mind, a couple of little notes include:
- usage of jefkel as test accounts, I am unsure if he is permanently off the project and if these are strictly linked to a test capacity.
- Mixed usage of yaml and yml, seems like almost a 50/50 split, I am unsure if it would be worthwhile to maintain a consistent pattern throughout.
Otherwise, everything seems to look good and make sense.

**Comment by caggles** [comment]
1. Yes, those are just test accounts, I'm not worried about them.
2. I hadn't noticed the mixed use of yaml and yml, I'll fix!

**Review by j-pye — APPROVED** [review]
I think this looks good. I only skimmed but the logic looks solid as far as I can tell without doing a bunch of manual tests. And I believe you've already done a bunch of testing so I think you're good to merge 🤠

**Comment by caggles** [comment]
I'm just gonna fix the inconsistent yaml/yml and do another quick test to make sure i didn't break anything in the process, lol. Then I'll merge and we'll FINALLY be able to push it into silver! Yay!

**Pull request merged**